README.md

```diff
@@ -47,100 +44,13 @@ This module automates the install and configuration of Prometheus monitoring too
 * Installs a configuration file for prometheus daemon (/etc/prometheus/prometheus.yaml) or for alertmanager (/etc/prometheus/alert.rules)
 * Manages the services via upstart, sysv, or systemd
 * Optionally creates alert rules
-* The following exporters are currently implemented: node_exporter, statsd_exporter, process_exporter, haproxy_exporter, mysqld_exporter, blackbox_exporter, consul_exporter, redis_exporter, varnish_exporter, graphite_exporter, postgres_exporter, collectd_exporter, grok_exporter, ipsec_exporter, openldap_exporter, openvpn_exporter, ssh_exporter, ssl_exporter
-
-## Usage
-
-**Notice about breaking changes**
-
-Version 5.0.0 and older of this module allowed you to deploy the prometheus server by doing a simple `include prometheus`.
-We introduced a [new class layout](https://github.com/voxpupuli/puppet-prometheus/pull/194) in
-version 6. By default, including the `prometheus` class won't deploy the server now.
-You need to include the `prometheus::server` class for this (which has the same
-parameters that `prometheus` had). An alternative approach is to set the
-`manage_prometheus_server` parameter to true in the `prometheus` class. Background information about this change is described in the related [pull request](https://github.com/voxpupuli/puppet-prometheus/pull/187) and the [issue](https://github.com/voxpupuli/puppet-prometheus/issues/184).
-
-For more information regarding class parameters please take a look at the class docstrings.
+* The following exporters are currently implemented: apache_exporter, beanstalkd_exporter, bind_exporter, bird_exporter, blackbox_exporter, collectd_exporter, consul_exporter, dellhw_exporter, elasticsearch_exporter, graphite_exporter, grok_exporter, haproxy_exporter, ipmi_exporter, ipsec_exporter, jmx_exporter, memcached_exporter, mesos_exporter, mongodb_exporter, mysqld_exporter, nginx_prometheus_exporter, nginx_vts_exporter, node_exporter, openldap_exporter, openvpn_exporter, php_fpm_exporter, postfix_exporter, postgres_exporter, process_exporter, puppetdb_exporter, rabbitmq_exporter, redis_exporter, snmp_exporter, ssh_exporter, ssl_exporter, statsd_exporter, systemd_exporter, unbound_exporter, varnish_exporter, wireguard_exporter,
```

Comment:
should we just drop it and tell the people to check the REFERENCE.md? That lists all classes/exporters? We will always forget to update the README.md.

Reply:
fine with me